repos/fdbt-site/src/pages/ticketConfirmation.tsx

```diff
@@ -237,6 +238,69 @@ export const buildPointToPointPeriodConfirmationElements = (ctx: NextPageContext
     return confirmationElements;
 };
 
+export const buildFlatFarePriceByDistanceConfirmationElements = (
+    ctx: NextPageContextWithSession,
+): ConfirmationElement[] => {
+    const confirmationElements: ConfirmationElement[] = [];
+    const serviceInformation = getSessionAttribute(ctx.req, SERVICE_LIST_ATTRIBUTE) as ServiceListAttribute;
+    const services = serviceInformation ? serviceInformation.selectedServices : [];
```

Comment:
We shouldnt need to check if its there, as we're confident enough to cast it to ServiceListAttribute above.